packages/frontend/src/components/Avatar.tsx

```diff
@@ -0,0 +1,30 @@
+import { Tag } from '@blueprintjs/core';
+import React from 'react';
+import { useApp } from '../providers/AppProvider';
+
+type AvatarProps = {
+    initials: string;
+};
+
+export const Avatar: React.FC<AvatarProps> = ({ initials }) => (
+    <Tag
+        round
+        large
+        interactive
+        style={{
+            width: '30px',
+            padding: '5px 0px',
+            textAlign: 'center',
+        }}
+    >
+        {initials.substr(0, 2)}
```

Comment:
Should we uppercase ? users can type their names in lowercase. 
```suggestion
        {initials.substr(0, 2).toUpperCase()}
```